src/Induction/WellFounded.agda

```diff
@@ -79,6 +89,26 @@ module All {a lt} {A : Set a} {_<_ : Rel A lt}
   Please use wfRecBuilder instead."
   #-}
 
+module FixPoint
+  {a ℓ} {A : Set a} {_<_ : Rel A ℓ} (wf : WellFounded _<_)
+  (P : Pred A ℓ) (f : WfRec _<_ P ⊆′ P)
+  (f-ext : (x : A) {IH IH' : WfRec _<_ P x} → (∀ y y<x → IH y y<x ≡ IH' y y<x) → f x IH ≡ f x IH')
+  where
+
+  someWfRecInv : ∀ x → (q q' : Acc _<_ x) → Some.wfRec P f x q ≡ Some.wfRec P f x q'
+  someWfRecInv = All.wfRec wf (a ⊔ ℓ)
+                           (λ x → (q q' : Acc _<_ x) → Some.wfRec P f x q ≡ Some.wfRec P f x q')
+                           (λ { x IH (acc rs) (acc rs') → f-ext x λ y y<x → IH y y<x (rs y y<x) (rs' y y<x) })
+
+  open All wf ℓ
+  wfRecFixpLemma : ∀ {x} y y<x → wfRecBuilder P f x y y<x ≡ wfRec P f y
```

Comment:
If we have `FixPoint` in the module name we probably don't need `fixp` in the internal names? As the library is nearly entirely made up of lemmas we also try and avoid `lemma` in the name otherwise it'd be everywhere! Maybe this should be called `wfRecBuilder-wfRec` as it relates the two of them? Open to other suggestions though?

Reply:
I think `wfRecBuilder-wfRec` is indeed a more sensible name. I will make the change.
What about `wfRecFixpoint` (previously `wfRecFixp`)? Should I somehow get rid of `Fixpoint` there as well?

Reply:
I like the `unfold-wfRec` which you've changed it to.